neqo-http3/src/connection.rs

```diff
@@ -53,6 +53,22 @@ where
     pub priority: Priority,
 }
 
+pub enum WebTransportSessionAcceptAction {
+    Accept,
+    Reject,
+    Redirect
```

Comment:
I think we might want to treat `Redirect` as `Reject`, so maybe change this to:
```
pub enum WebTransportSessionAcceptAction {
    Accept,
    Reject(Vec<Header>)
}
```

Reply:
It might be OK to signal redirects separately, but we'd also want to know *where* the redirect was to.